test/test_utils.py

```diff
@@ -1,54 +1,110 @@
 import pytest
+import numpy as np
 from numpy.testing import assert_allclose
 
-from report_data import ReportData
-from utils import summarize_report
+from pyfifotax.report_data import ReportData
+from pyfifotax.utils import summarize_report
 
 
-def get_elster_summary(file_name, year, mode):
-    report = ReportData(sub_dir="test/files", file_name=file_name)
+def get_elster_summary(
+    file_name,
+    year,
+    mode,
+    apply_stock_splits=True,
```

Comment:
I recommend to enumerate real-life stock splits in the example spreadsheet (even for multiple companies if you would like to have more entries). It took me some seconds to realise that the stocks were not in the right amount and for this reason the calculation was off.

It's possible that not everyone will open and check the file.

Reply:
What do you mean exactly? Printing out that stock-splits are applied?

Reply:
Still don't what you mean with this one.